packages/gatsby/src/utils/babel-loader-helpers.js

```diff
@@ -48,95 +48,14 @@ const prepareOptions = (babel, resolve = require.resolve) => {
     )
   }
 
-  // Fallback presets/plugins
+  // Fallback preset
   const fallbackPresets = []
-  const fallbackPlugins = []
-
-  let targets
-  if (stage === `build-html`) {
-    targets = {
-      node: `current`,
-    }
-  } else {
-    targets = {
-      browsers: pluginBabelConfig.browserslist,
```

Comment:
I'm a bit confused here about `pluginBabelConfig.browserslist`. Where are we reading `pluginBabelConfig` from? There seems to be a `./.cache/babelState.json` file but I don't think I can access this from our new npm package.

Reply:
See https://github.com/gatsbyjs/gatsby/pull/8724#discussion_r224992947